.github/workflows/update-sdk.yml

```diff
@@ -0,0 +1,27 @@
+name: update-dotnet-sdk
+
+on:
+
+  # Scheduled trigger to check for .NET SDK updates at 12 UTC every Monday
+  schedule:
+    - cron:  '00 12 * * MON'
+
+  # Manual trigger to update the .NET SDK on-demand.
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  issues: write
+  pull-requests: write
+
+jobs:
+  update-dotnet-sdk:
+    name: Update .NET SDK
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: martincostello/update-dotnet-sdk@31f3d5a6c2c4a5a0b7eb0e0d5c30501925d6e790
+      with:
+        quality: 'daily'
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        branch-name: dev
```

Comment:
@wtgodbe This isn’t right - you need to tell the checkout action to clone the dev branch instead of telling this action to create a branch called dev.